util/src/main/java/tc/oc/pgm/util/math/TransformMatrix.java

```diff
@@ -0,0 +1,81 @@
+package tc.oc.pgm.util.math;
+
+import org.bukkit.util.Vector;
+
+public class TransformMatrix {
```

Comment:
I'll trust you on this one. I don't want to revisit linear algebra to this extent